flow/instrumentation.h

```diff
@@ -46,6 +42,9 @@ class Stopwatch {
   fml::TimePoint start_;
   std::vector<fml::TimeDelta> laps_;
   size_t current_sample_;
+
+  double one_frame_ms_;
```

Comment:
Here and elsewhere, `std::chrono::duration`

Reply:
This seems to be a big change as all the computations in instrumentation.cc needs to be updated to use `std::chrono::duration` instead of double/float. I'd prefer to do that in another PR to not pollute the diff.

Reply:
Now I stored it as `std::chrono::duration`. But I converted it to `one_frame_ms` in the computations so I don't have to make too many changes.

Reply:
Sounds good.